srcStatic/Log/LoggerDebug.cpp

```diff
@@ -2,8 +2,9 @@
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 
+const std::string newline("\r\n");
 
 void LoggerDebug::Log(const std::string& message)
 {
-	OutputDebugString(message.c_str());
+	OutputDebugString((message + newline).c_str());
```

Comment:
Question, was this change to make output function *code* more symmetric? (Without regard to actual *displayed* output?) If that's the case, I would recommend no change.

I'm unclear on whether or not  `OutputDebugString` provides automatic line termination. It seems likely a function like that would cause messages to be automatically separated by lines. I suppose it might depend on the attached debugger though.

Given that we're having this discussion though, I assume actual output alerted you to the difference (as opposed to source code asymmetry), and that `OutputDebugString` does not result in automatic newlines. If that is the case, then I would suggest:
```suggestion
	OutputDebugString((message + "\n").c_str());
```

That should be quite understandable to the majority of developers, will little syntactic noise, and no cross referencing of definitions.

Reply:
OutputDebugString does not line terminate provided input. I'll push your suggestion momentarily.